eo-maven-plugin/src/main/java/org/eolang/maven/rust_project/Cargo.java

```diff
@@ -60,13 +59,21 @@ public class Cargo {
      * @param name Name of lib.
      */
     public Cargo(final String name) {
-        this.pack = ImmutableMap.of(
-            "name", name,
-            "version", "0.1.0",
-            "edition", "2021"
+        this.pack = Stream.of(
```

Comment:
@levBagryansky Looks progressive :)
But maybe plain old java approach will look simpler:
```java
this.pack = new HashMap();
this.pack.put("name", name);
this.pack.put("version", "0.1.0");
...
```
What do you think?

Reply:
@volodya-lombrozo No(. We have a check forcing to only initialize fields in Ctor.

Reply:
@volodya-lombrozo We discussed it in #1945

Reply:
@levBagryansky What about [prestructors](https://www.yegor256.com/2021/08/04/prestructors.html)?  Could they help us here?

Reply:
@volodya-lombrozo sounds cool